sig-cluster-lifecycle/grooming.md

```diff
@@ -0,0 +1,37 @@
+# SIG Cluster Lifecycle Grooming
+
+## Overview 
+
+The purpose of this document is to outline the current backlog grooming practices, or process, that are employed across several sub-projects under SIG Cluster Lifecycle.  By providing a semi-consistent set of best practices it allows developers to work across sub-projects fairly easily.  More importantly it signals direction and priority to developers who may be new to the sub-projects.  
+
+This document is an overview of the practices that are currently followed and any feedback, or refinement, is solicited. 
+
+## Process 
+
+### Planning a milestone
+For every sub-project under sig-cluster lifecycle, we highly recommend holding a planning session prior to beginning a new milestone.  The purpose of planning is to outline a set of high level deliverables that the community has decided that they want to work on for this upcomming milestone.  
+
+### Creating milestones
+Now that you have a plan, you're about to embark upon a new journey, let's call this journey `1.X` on a fresh repo.  The first step is to create two milestones (`1.X`, and `Next`).  The reason for the `Next` milestone is simply to create a placeholder to segregate work items out of the current milestone.  We typically tr to stick with just two milestones, because we have found over time that this simplfies execution and keeps us focused on trying to deliver for the current milestone.
```

Comment:
`We typically try` <- missing `y` in `try`

Reply:
Fixed